Public/Src/Engine/Dll/Distribution/RemoteWorker.cs

```diff
@@ -214,11 +219,30 @@ private void SendBuildRequests()
             }
         }
 
-        public void LogExecutionBlob(ArraySegment<byte> executionLogBlob, int blobSequenceNumber)
+        [SuppressMessage("AsyncUsage", "AsyncFixer02:awaitinsteadofwait")]

```

Comment:
>SuppressMessage [](start = 9, length = 15)

In other places we use a fully qualified name (ie.., System.Diagnostics.CodeAnalysis.SuppressMessage), should we make it consistent?